_posts/2018-07-31-craft-boilerplate-for-heroku-and-local.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Craft 3 Heroku Boilerplate'
+author: dimitri-steinel
+layout: post
+mainImage: ""
+mainImageAlt: ""
+--- 
+
+### First contentful paint
+To make sure we can seamlessly work in teams, we have a specific requirements for our hosting environment.
+Currently, our hosting solution of choice is [Netlify](https://www.netlify.com/) and [Heroku](https://www.heroku.com/). 
+Netlify is mostly for our static websites. Heroku is for almost all other projects.
+With [Craft 3](https://craftcms.com/news/craft-3), which was released in April’18, it is already much easier to install the CMS as [composer package](https://packagist.org/packages/craftcms/cms) and host it. But there were still some things we need to take care of in order to host it on Heroku.
+
+Here is a guide on how to set up a Craft 3 project on Heroku.
+
+### The quick and dirty start
+This button has been made for those who just want to have a fresh Craft 3 instance on Heroku. Click the button and directly deploy our public [boilerplate repository](https://github.com/edenspiekermann/craft-heroku-boilerplate) to a new instance on your Heroku account. 
+
+Go ahead and try:
+[![Deploy](https://www.herokucdn.com/deploy/button.svg){:class="img--little-margin"}](https://heroku.com/deploy?template=https://github.com/dsteinel/craft-heroku-test-project/tree/master)
+
+After the install process, visit the URL of your app and add `/admin/install` to it. For example: 
+`https://HERE-GOES-YOUR-URL.herokuapp.com/admin/install`
+
+Fill out all the details from the guide and you are good to go!
+![Heroku create new app screen](https://res.cloudinary.com/dsteinel/image/upload/v1533126210/Espi/blog/heroku-craft/2018-08-01_14.21.56.gif){:class="img--little-margin"}
+
+
+## Start from scratch
+### Set up a new project
+First we need to clone our Boilerplate Repo into a local folder:
+{% highlight bash %}
+  $ git clone git@github.com:edenspiekermann/craft-heroku-boilerplate.git
+{% endhighlight %}
+
+Rename the folder and make it your project:
+{% highlight bash %} 
+  $ mv craft-heroku-boilerplate NEW-PROJECT-NAME
+{% endhighlight %}
+
+Remove the Github connection and connect the folder with your own Git repository. Go and create a new and empty Git repository. There must not be any `.git` or `README.md` file. After it is created, go with:
```

Comment:
could you just do a shallow clone instead?
```bash
$ git clone git@github.com:edenspiekermann/craft-heroku-boilerplate.git --depth 0
```
?

Would that work better than deleting the .git folder?